### DISCUSSION TITLE
Question on available icons

### DISCUSSION THREAD

**Discussion by Zabooby** [excalith/excalith-start-page#35] [Q&A] [answered]
Are icons from [this ](https://iconify.design/) site the only ones that can be used?

**Comment by excalith** [comment, marked as answer]
Hey @Zabooby, iconify is the only icon provider for this project and it has lots of styles and icons to choose from. However, on your fork you can change the icon provider as you want. In order to do that, you need to change several stuff:
- Add new icon package through yarn
- Remove iconify package through yarn
- Refactor [Link.js](https://github.com/excalith/excalith-start-page/blob/main/src/components/Link.js) component
- Change icon source strings on your data

Please don't forget to mark this post as answered.
Cheers!
